### DISCUSSION TITLE
what library will we use to read the different flags value?

### DISCUSSION THREAD

**Discussion by Junaid300** [khattakdev/minimalcommit#18] [General]
I am not sure what is exact word for the flags but i am talking about (`mct --help`) `--help part`. @khattakdev  do you have any library in your mind to read the  values other than mct? Or will we go for built-in feature of nodejs?

**Comment by khattakdev** [comment]
Use [yargs](https://www.npmjs.com/package/yargs)

Side note: A better place to ask such question is the Issues tab, under the specific issue
